app/src/main/java/net/squanchy/speaker/SpeakerDetailsActivity.kt

```diff
@@ -66,14 +69,17 @@ class SpeakerDetailsActivity : AppCompatActivity() {
         subscriptions.add(
             service.speaker(speakerId)
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe { onSpeakerRetrieved(it) }
+                .subscribe(
+                    ::onSpeakerRetrieved,
+                    { e -> Timber.e(e, "Error retrieving the speaker") }
```

Comment:
`Timber.e(it, ...)`

Reply:
No, I think it's more readable this way.